tests/config/test_keys.py

```diff
@@ -142,7 +142,7 @@ def test_display_key_for_urwid_key(urwid_key: str, display_key: str) -> None:
 
 
 COMMAND_TO_DISPLAY_KEYS = [
-    ("NEXT_LINE", ["Down", "Ctrl n"]),
+    ("SEND_MESSAGE", ["Ctrl d", "Meta Enter"]),
```

Comment:
I'd have the same question :)

Reply:
Also, partially answering it myself. The commit message gives it away: The change seems to aim to

> not use readline hotkeys [in tests]

but I'm not sure *why*. Is it because we can not _rely_ on the existence of readline shortcuts due to upstream changes? That seems reasonable to me. In general, it could be helpful to add the motivation for the goal to the commit message :)

Reply:
I forgot to get back regarding this, @zormit, my apologies.
Sashank and I had talked about this, so I hadn't added an answer here for his initial question.

Yes, you're spot on.
Thank you for the pointer. You've worded it pretty well, I did not think of it in those terms.